### PR TITLE
integrals: simplified convergence conditions

### PR DESCRIPTION
**Pull request by jksuom**
#### Brief description of what is fixed or changed
Allow `arg` in `periodic_argument`.

`periodic_argument` does currently leave unevaluated all expressions that
contain `periodic_argument`, `atan2`, `arg` or `atan` after simplifications.
Therefore expressions of the following type will often appear in the
 convergence conditions of integrals computed by meijerint.

    >>> unbranched_argument(1/polar_lift(x))
    periodic_argument(1/polar_lift(x), oo)
These expressions will be more readable if `periodic_argument`
is allowed to return values containing `arg`.
    
    >>> unbranched_argument(1/polar_lift(x))
    -arg(x)

#### Other comments
Some conditions are currently simplified in `transforms` and
`meijerint` modules. The pattern tables are extended for `arg`
in order to maintain the simplified output format.

### CODE REVIEW

**Comment by smichr** [comment]
Looks ok to me.

File: sympy/integrals/meijerint.py [hunk 1]
```diff
@@ -604,6 +604,10 @@ def _condsimp(cond):
         (Or(p < q, Eq(p, q)), p <= q),
         # The next two obviously are instances of a general pattern, but it is
```
Comment (normalhuman) [inline sympy/integrals/meijerint.py]: It's not "the next two" anymore, I count four

Comment (jksuom) [reply sympy/integrals/meijerint.py]: Well, I think it as two for `arg`, two for `unbranched_argument`.

Comment (jksuom) [reply sympy/integrals/meijerint.py]: It may turn out that the latter pair can be removed.

**Comment by asmeurer** [comment]
Should periodic_argument itself be changed here?

**Comment by jksuom** [comment]
I tried not to change too much at once. The object was to only make debugging easier with more readable conditions.

**Pull request merged**